spec/models/dimensions/item_spec.rb

```diff
@@ -39,6 +39,32 @@
         expect(item.get_content).to eq("Lorem ipsum dolor sit amet.")
       end
 
+      it "returns content json if schema is 'licence'" do
+        json = { schema_name: "licence",
```

Comment:
This looks good but I think with these tests it's easy to test a JSON payload that isn't what the publishing API actually returns. 

There is a test helper called `GovukSchemas::RandomExample` which can help avoid this - it generates a random example that is validated against the schemas, so if you use that we have more confidence that all the field names match up.

It's a bit awkward to do because there are different schemas for notifications, publishing, and content store. Some existing examples:
- https://github.com/alphagov/email-alert-service/blob/0a3ed2701816a3aa11ed0639e31aea8ff2cfb196/spec/integration/random_input_spec.rb
-
 https://github.com/alphagov/rummager/blob/1b3d7d2fb4e5c13c1e17e6493150ef77ab12b349/spec/support/spec_helpers.rb#L40

Reply:
Good point @MatMoore, happy to add integration tests in a similar way you have done in `email-alert-service`. I have collected your comment to address it in this quarter.